go-controller/pkg/ovn/master.go

```diff
@@ -831,6 +850,16 @@ func (oc *DefaultNetworkController) deleteNodeEvent(node *kapi.Node) error {
 	if err := oc.deleteNode(node.Name); err != nil {
 		return err
 	}
+
+	chassisID, err := util.ParseNodeChassisIDAnnotation(node)
+	if err != nil {
+		return err
```

Comment:
return a proper `fmt.Errorf` saves lots of time during escalations.. helps narrow down the percolated error easier.

Reply:
Actually I think we should just conditionally call DeleteChassisTemplateRecord() if ParseNodeChassisIDAnnotation() succeeded.  I did that instead.